content/_data/roadmap/roadmap.yml

```diff
@@ -46,6 +46,11 @@ categories:
       description: Provide out-of-the-box support for manual and automatic promotion of build artifacts in a separate Pipeline after the job completion
       status: future
       link: https://issues.jenkins-ci.org/browse/JENKINS-36089
+    - name: Improve Pipeline step documentation generator
+      status: future
+      description: >
+        Enhance the Jenkins Pipeline documentation generator to produce better documentation for thousands of Pipeline developers.
+      link: https://www.jenkins.io/projects/gsoc/2020/project-ideas/pipeline-step-documentation-generator/
```

Comment:
I was under the impression that we did not want fully qualified https links but rather the shortest relative link possible. I could be wrong.

Reply:
It is a special case, a resource YAML file which is an open data file for the project. Having absolute URLs helps potential consumers. But I do not have strong opinion